### PR TITLE
Rename .json files after being processed

### LINKED ISSUE #104: Rename processed json files
The API file watcher may run into a limit (~30,000 files) if we keep the old .json files around. So instead of watching them after they have been processed, rename them to an extension that it not being watched.

### PR DESCRIPTION
**Pull request by brdunfield**
Add a .processed extension to the end of json files once they have been processed.

Closes #104

### CODE REVIEW

File: api/src/utils/renameFile.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as fs from "fs";
```
Comment (timarney) [inline api/src/utils/renameFile.ts]: We should be able to pull in just fs here
```
const fs = require("fs");
```

File: api/src/utils/renameFile.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export const renameFile = async (filePath: string) => {
+  // assume file ends with .json
+  let newFilePath = filePath.slice(0, -5) + ".processed";
+  fs.rename(filePath, newFilePath, function(err) {
```
Comment (timarney) [inline api/src/utils/renameFile.ts]: Minor but let's make this an arrow function to keep things consistent
```
(err) => {
    if(err) throw err;
  } 
```

File: api/src/watcher/index.ts [hunk 3]
```diff
@@ -30,11 +31,13 @@ export const saveWatchedFile = async (path: string) => {
   const data = JSON.parse(file);
   globalQueue.push(data, () => {
     console.log(`finished processing ${path} ${counter}`);
+    // add .processed to the end of the filename
+    renameFile(path);
```
Comment (timarney) [inline api/src/watcher/index.ts]: Thinking we should wrap a try / catch around the 
```
renameFile(path);
```

**Comment by brdunfield** [comment]
I will add renaming existing files to this PR and let you know when it's ready.

File: api/src/db/getFiles.ts [hunk 4]
```diff
@@ -27,6 +28,16 @@ export const getFiles = async (path: string = process.env.CHECKS_PATH) => {
       return readFile(file);
     }),
   );
+    /*
```
Comment (timarney) [inline api/src/db/getFiles.ts]: Can you remove this then we're all set.

**Pull request merged**